_includes/markdown/Systems.md

```diff
@@ -201,54 +201,71 @@ The three commonly encountered MySQL replication types are Master-Slave, Master-
+
+<h2 id="https" class="anchor-heading">HTTPS{% include Util/top %}</h2>
+
+New sites should implement HTTPS (Hypertext Transfer Protocol Secure) unless there is a specific reason not to, even if they don't handle sensitive communications. HTTPS provides many benefits over HTTP such as:
+
+* **Security:** HTTPS creates an encrypted channel between your browser and a website's server. Without encryption, anything transferred between the website and browser could be intercepted by a 3rd party. This is an issue for usernames, passwords, payment information and anything else sent over a form.
+
+* **Performance:** The HTTP/2 web protocol includes numerous performance benefits such as allowing multiple requests to be sent over a single connection, which only works on websites running on HTTPS.
+
+* **SEO:** While Google's search algorithms are ever-evolving, [Google is moving towards making sure that HTTPS sites will get a page rank boost](https://webmasters.googleblog.com/2014/08/https-as-ranking-signal.html). Using HTTPS automatically gives sites ranking improvements over non-HTTPS websites.
+
+* **Browser Compatibility:** Modern browsers such as Google Chrome and Firefox display a 'Not Secure' message on any page where data is entered into a form that isn't protected by HTTPS. This is a poor experience for users that could result in an increased bounce rate.
+
+#### **HTTPS Best Practices**
+* **Mixed Content** - Mixed content occurs when HTML is loaded over a secure HTTPS connection and requests resources such as images and scripts to be loaded over an insecure HTTP connection. Most browsers display warnings about this type of content and prevents this content from loading on a page. Before implementing a new website on HTTPS, it's important to run a mixed content scanning tool that will crawl and scan an HTTPS-enabled website for mixed content. WordPress plugins should also be utilized that will check for insecure content from within the WordPress editor interface.
+
+* **Non-HTTPS APIs** - Caution should be taken when sending sensitive information such as usernames or passwords to 3rd-party APIs not utilizing HTTPS. When sending data to a non-secure API, always be sure sensitive information is encrypted and decrypted on each end of the connection so that they are not exposed. Likewise, sensitive information, such as passwords, session tokens, etc should not be exposed in HTTP URL requests, which can be captured in web server logs.
```

Comment:
"sensitive information, such as passwords, session tokens, etc should not be exposed in HTTP URL requests, which can be captured in web server logs" - these can be captured in web server logs even when the request is over HTTPS as HTTPS will be terminated on the web server.  What are we trying to say here?

Reply:
I removed this section, as I don't think it necessarily belongs. I think originally this was supposed to be warning people not to pass information like passwords to non-HTTPS requests like in the POST data because it's not encrypted. I'd be more interested in getting this out and we can always add it back in later on.